README.md

```diff
@@ -61,86 +61,30 @@ The official home of skills for the Mycroft ecosystem.  These skills are written
-```$name-your-skill``` is the name used to install it via MSM or "Hey Mycroft, install ...".  The recommended format for skill names is "publisher-descriptive-name", where 'publisher' is a unique name for you or your organization.  For example, "penrod-nautical-speed-translator".
-
-When picking a name keep in mind that the installer will match by whole words between the dashes.  So if a user says
-"install speed translator" it will look for all skills in the repo with the words 'speed' AND 'translator'.  That
-means it will find "penrod-nautical-speed-translator" but would not find "abc-nautical-speeds-translator".  Make
-sure the pieces of the name are 'speakable' to allow verbal installs.  That means "fubar-v2timer" would be a bad
-name since you can't speak "v2timer" as a word.  A better name would be "fubar-timer-v2" or "fubar-timer-version-2".
-
-The above command should have modified the .gitmodules file and added something similar to the bottom of the file:
-```
-+[submodule "NAME-OF-YOUR-SKILL"]
- +	path = YOUR-SKILL-REPO (or any unique path withing the mycroft-skills repo)
- +	url = https://github.com/USERNAME/YOUR-SKILL-REPO.git
-```
-
-For more help, feel free to check out this [guide to working with submodules](https://github.com/blog/2104-working-with-submodules)
-
-### 5) Modify this README.md
-Modify the table section below to include the direct link to your repo.  Including the break HTML tag and an example phrase
-or two that trigger your skill:
-
-```
-| :heavy_check_mark:  | [home-assistant](https://github.com/btotharye/mycroft-homeassistant#readme)| Control your devices in home-assistant<br>```turn on office``` |
-
-```
-Chose an appropriate status icon from the list below:
-
-**Status:**  
-:heavy_check_mark: good working order  
-:construction:     still being developed and not ready for general use (for reference/collaboration)  
-:question:         untested (by us)  
-:skull:            Broken, but good for ideas!
-
-### 6) Submit a PR (Pull Request)
-Once you've got your local version of the repo organized properly, submit a PR.
-
-### MSM Compliance
-To make your skill capable of being installed via MSM (the Mycroft Skill Manager) you can include two additional files.
-
-##### requirements.txt
-A list of all Python modules which must be installed for it to work.  These will be installed via the Python PIP utility
-
-##### requirements.sh
-A script to run that will perform any additional steps needed to prepare the system for your skill.  This can include package installations.
+* [Introduction to Developing Skills for Mycrof](https://mycroft.ai/documentation/#introduction-to-developing-skills-for-mycroft)
+* [Getting started with Mycroft Skills](https://mycroft.ai/documentation/#developing-skills-for-mycroft)
+* [Skill Settings](https://mycroft.ai/documentation/#skill-settings)
+* [Skills Acceptance Process](https://mycroft.ai/documentation/#skills-acceptance-process)
```

Comment:
Should there be a couple of examples of the upload process?

Something quite basic along the lines of

To upload a skill
`msk upload PATH/TO/SKILL`

To update an existing skill:
`msk update PATH/TO/SKILL`

Reply:
Good question. I had considered putting a few basic commands in this section, however `msk` requires installation - if someone is reading the README.md then I think it is better they read the `msk` documentation, rather than say trying the `msk` command and it not working.